HW2/P5/driver.py

```diff
@@ -80,3 +103,4 @@ def randcolor():
             # SUBPROBLEM 3: sort objects by location.  Be sure to update the
             # grid if objects' indices change!  Also be sure to sort the
             # velocities with their object positions!
+            # at first, we create a
```

Comment:
I am positive I did. It's the subproblem 3 using z-order...It's weird..let me look for it.